scripts/Modules/mkw_utils.py

```diff
@@ -1,14 +1,53 @@
 from dolphin import memory, utils
 
-from .mkw_classes import mat34, quatf, vec3, ExactTimer
-from .mkw_classes import VehicleDynamics, VehiclePhysics, RaceManagerPlayer
+from .mkw_classes import mat34, quatf, vec3, ExactTimer, eulerAngle
+from .mkw_classes import VehicleDynamics, VehiclePhysics, RaceManagerPlayer, KartObjectManager, RaceManager, RaceState
+
+import math
 
 # These are helper functions that don't quite fit in common.py
 # This file also contains getter functions for a few global variables.
 
 # NOTE (xi): wait for get_game_id() to be put in dolphin.memory before clearing
 #  these commented-out lines:
 
+
+class FrameData:
```

Comment:
Could this potentially conflict with `agc_lib.py`?

Reply:
It can conflict if you import both class with the same name.
Currently it's not an issue, but if soemone makes some script using both agc_lib and mkw_utils, it could be.
I'll eventually merge both class into the same class (currently, agc_lib.FrameData store values you can read in the memory, and mkw_utils.FrameData is more messy, but can store function from values you can read in the memory, and only used for the draw_info_display.)